src/content/team/TeamCard.test.js

```diff
@@ -53,4 +55,15 @@ describe('<TeamCard>', function() {
         let noEmail = {...testTeam, emails: undefined}
         render(<TeamCard {...noEmail} />)
     })
+    it('should open and close the dialog', function() {
+        render(<TeamCard {...testTeam} />)
+        let openButton = screen.getByRole("button", { name: "See All Projects"})
+        userEvent.click(openButton)
+        let dialog = screen.getByRole("dialog")
+        expect(dialog).not.toBeNull()
+        let closeButton = screen.getByRole("button", <CloseIcon/>)
+        userEvent.click(closeButton)
+        let dialogAgain = screen.queryByRole("dialog")
+        expect(dialogAgain).toBeFalsy
```

Comment:
`.toBeFalsy` -> `.toBeFalsy()`
https://jestjs.io/docs/expect#tobefalsy
Although `.toBeNull()` is a more accurate call because `queryByRole` returns `null` on no matches.
https://testing-library.com/docs/queries/about/

Reply:
I just pushed my code, should be good now. `toBeNull()` -> `.not.toBeVisible()`